README.md

```diff
@@ -97,10 +97,13 @@ This JSON file will generate the following Kinesis Record
 | -p, --partition-key <value> | The partition key | No | 1 |
 | -e, --localstack-endpoint <value> | The localstack endpoint | No | http://localhost:4566 |
 | -o, --operation <value> | The operation* name you want to simulate | No | LOAD |
+| -c, --chunk-size <value> | chunk-size for batch processing | No | 1 |

```

Comment:
I don't think chunk-size is a good name for this parameter. Chunk size is usually related to the size in bytes of something. This is actually the batch size, which is: how many records you want to send in a single batch (something between 1 and 500). AWS also does not uses the term chunk-size when talking about batches, they either use batch size or number of records.

Reply:
Yep, makes total sense, just updated the parameter to batch-size

Reply:
Awesome ! Can you get update your branch to contain the latest code ? I'll merge the PR after that

Reply:
I mean, get the latest code from develop and fix conflicts. Please keep the yarn.lock conflits from the origin (my develop), because I updated several libraries that had vulnerabilities

Reply:
Done! Can you check it? 😄

Reply:
Just a question. Since this is a big refactor, will this release be a major or a minor? Just asking because of the wording in the commit.

Reply:
It’s a minor because it’s not a breaking change. Just changed the underlying implementation and added a new parameter.